app/models/changelog_admin/null_referenceable.rb

```diff
@@ -0,0 +1,15 @@
+module ChangelogAdmin
+  class NullReferenceable
+    def icon
+      "https://https://via.placeholder.com/150"
```

Comment:
This is the default icon when nothing is referenced in the changelog entry.

Reply:
OK - cool. I'll work this out with Nic when we do styling.